CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/7.0.0...master)
 ### Backward Compatibility Breaks
 ### Added
-
+Ability to specify the type of authentication manually by the `auth_type` parameter (in the client class config) was added (allowed values are `basic, digest, gssnegotiate, ntlm`)
```

Comment:
Nit: Sorry, didn't spot this before, but could you prefix it by `* ` to make it a list? Same for line 16.

Reply:
Oh you are right
I changed it.